ui/src/app/views/notebook-redirect/component.html

```diff
@@ -38,5 +38,5 @@ <h2 class="notebooks-header" *ngIf="creating">Creating New Notebook '{{notebookN
 </div>
 <div *ngIf="progress === Progress.Redirecting || progress === Progress.Loaded" [class.hidden]="progress === Progress.Redirecting" class="i-frame">
   <div id="seperator" *ngIf="progress === Progress.Loaded"></div>
-  <iframe id="leo-iframe" style="width:100%; height: 33rem;" frameBorder="0" [src]="leoUrl"></iframe>
+  <iframe id="leo-iframe" style="width:100%" frameBorder="0" [src]="leoUrl"></iframe>
```

Comment:
Does this work for you? I pulled your branch and it does not seem to be working.

![image](https://user-images.githubusercontent.com/822298/48282871-c37a1380-e40f-11e8-9438-1b858530ddb1.png)